tests/tests/control_atmwav

```diff
@@ -48,8 +48,14 @@ export LIST_FILES="sfcf000.nc \
                    RESTART/sfc_data.tile4.nc \
                    RESTART/sfc_data.tile5.nc \
                    RESTART/sfc_data.tile6.nc \
-                   out_grd.glo_1deg \
-                   out_pnt.points"
+                   20210322.090000.out_grd.glo_1deg \
```

Comment:
Are these restart files at different hours or output? If they're restarts, we generally compare only the final restart.

Reply:
out_grd* files are gridded outputs, and out_pnt* files are point outputs.